qiskit/quantum_info/__init__.py

```diff
@@ -89,10 +89,7 @@
 
 """
 
-from .operators.operator import Operator
-from .operators.pauli import Pauli, pauli_group
-from .operators.channel import Choi, SuperOp, Kraus, Stinespring, Chi, PTM
-from .operators.measures import process_fidelity
+from .operators import *
```

Comment:
this is not a good practice since any irrelevant function that someone adds will also get imported at the top namespace in the future. It's good to be explicit about what the main imports are.